DomainContent/AvatarStore/Laundry/laundryZone.js

```diff
@@ -0,0 +1,15 @@
+//  laundryZone.js
+//
+//  Created by Rebecca Stankus on 9/25/17.
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  This script is applied to the dynamic laundry basket to keep it from moving when an item is thrown in.
```

Comment:
Just for clarity sake, can you update the description to detail that it's an edit reject filter that's applied to a surrounding zone for the basket?